### ISSUE TITLE
AttributeError: 'Request' object has no attribute '_body'

### ISSUE THREAD

**Issue by tianzhipengfei** [tiangolo/fastapi#4568]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
import uvicornapp = FastAPI()

@app.get('/chain_detail/')
def read_chain_detail():    
    return 'hello world'

if __name__ == '__main__':    
    uvicorn.run(app=app, host="127.0.0.1", port=8000)
```

### Description

I got "AttributeError 'Request' object has not attribute '_body'" error. As GET request doesn't have body, I have no idea how can I fix it.

### Operating System

Windows

### Operating System Details

Windows 10
Version No. 1809

### FastAPI Version

0.73.0

### Python Version

3.8.8

**Comment by tianzhipengfei** [comment]
> In my case there's no problem like you. Just try to install fastAPI by use `pip install "fastapi[all]`

I reinstalled FastAPI by the way you recommended, but I still got the error...

**Comment by upeshjindal** [comment]
Use uvicornapp in the decorator since that is your FastAPI app.

```Python
from fastapi import FastAPI
import uvicornapp = FastAPI()

@uvicornapp.get('/chain_detail/')
def read_chain_detail():    
    return 'hello world'

if __name__ == '__main__':    
    uvicorn.run(app=app, host="127.0.0.1", port=8000)
```

**Comment by tianzhipengfei** [comment]
> Use uvicornapp in the decorator since that is your FastAPI app.
> 
> ```python
> from fastapi import FastAPI
> import uvicornapp = FastAPI()
> 
> @uvicornapp.get('/chain_detail/')
> def read_chain_detail():    
>     return 'hello world'
> 
> if __name__ == '__main__':    
>     uvicorn.run(app=app, host="127.0.0.1", port=8000)
> ```

I don't think it matters, as it is just the name of the app. I tried to run your code, but it doesn't work. At least, the last line should be  `uvicorn.run(app=uvicornapp, host="127.0.0.1", port=8000)` . Unfortunately, After I modified it, I still got the error. Anyway, thanks for your help~

**Comment by upeshjindal** [comment]
Well, I think your original posted code has typing mistakes. I copied the code and ran it. It worked perfectly. See below:

```Python
from fastapi import FastAPI
import uvicorn # This line in your code just joined with the next line which is why I suggested to use uvicornapp in decorator.
app = FastAPI()


@app.get('/chain_detail/')
def read_chain_detail():
    return 'hello world'


if __name__ == '__main__':
    uvicorn.run(app=app, host="127.0.0.1", port=8000)
```

**Comment by upeshjindal** [comment]
Can you share how you are calling the endpoint.

**Comment by tianzhipengfei** [comment]
> Well, I think your original posted code has typing mistakes. I copied the code and ran it. It worked perfectly. See below:
> 
> ```python
> from fastapi import FastAPI
> import uvicorn # This line in your code just joined with the next line which is why I suggested to use uvicornapp in decorator.
> app = FastAPI()
> 
> 
> @app.get('/chain_detail/')
> def read_chain_detail():
>     return 'hello world'
> 
> 
> if __name__ == '__main__':
>     uvicorn.run(app=app, host="127.0.0.1", port=8000)
> ```

Oh, I got it, and modified it. Thanks for that. But it is not the point which caused the error. I just used chrome to visit http://127.0.0.1:8000/chain_detail/ . One more thing, I can see the correct response in chrome. But at the same time, I can see the error in console.
